include/sentry.h

```diff
@@ -1192,11 +1192,6 @@ SENTRY_API void sentry_remove_fingerprint(void);
  */
 SENTRY_API void sentry_set_transaction(const char *transaction);
 
-/**
- * Removes the transaction.
- */
-SENTRY_API void sentry_remove_transaction(void);
```

Comment:
this is intentional?

Reply:
Yeah, after talking with @relaxolotl on Slack. The method doesn't exist in other SDKs and doesn't do anything other than call `sentry_set_transaction(NULL)` anyway.